dhis-2/dhis-api/src/main/java/org/hisp/dhis/expression/ExpressionService.java

```diff
@@ -57,6 +57,7 @@
  * <li>Program data elements on the form D{programuid.dataelementuid}</li>
  * <li>Program tracked entity attribute on the form A{programuid.attributeuid}</li>
  * <li>Program indicators on the form I{programindicatoruid}</li>
+ * <li>Indicators on the form I{indicatoruid}</li>

```

Comment:
Perhaps this should be N{indicatoruid} ?

Reply:
Ops